added russian translate

v1docq47 opened the pull request:
added 2 files with translation "Monero Addresses Cheatsheet": 
- MoneroAddressesCS\translations\ru\MoneroAddressesCheatsheet20201206.docx
- MoneroAddressesCS\translations\ru\MoneroAddressesCheatsheet20201206.pdf

added [RU] links to the russian version of the translation of Mastering Monero and Zero to Monro: Second Edition

v1docq47 committed:
added russian translate

added 2 files with translation "Monero Addresses Cheatsheet": 
- MoneroAddressesCS\translations\ru\MoneroAddressesCheatsheet20201206.docx
- MoneroAddressesCS\translations\ru\MoneroAddressesCheatsheet20201206.pdf

baro77 commented:
Great job @v1docq47 ! Congrats, I love to see it in Cyrillic characters!

just before merging, let me say:
if you want to fix it, it seems there's a problem with arrows position in Stealth Addresses > Payee's POV;
and of course I don't know Russian, but I'm very curious and my educated guess is that a word is missing in the same row, rightmost cell, maybe "получатель"?! :)
Lastly, but not less important, I STRONGLY encourage you to add your (nick)name/link as translator in the free space under "ПРИМЕЧАНИЕ: ..." (as you know my first proposal -still valid- was to add your avatar near mine in the top left part, but I guess you don't like that idea... anyway I firmly think it would be fair for you to appear in the cheatsheet, at least in credits section)

PS
I was forgetting .. I would name the files
-  MoneroAddressesCheatsheet20201206-RU.docx
-  MoneroAddressesCheatsheet20201206-RU.pdf

to avoid problems in case of same folder storage (by third parties) with the originals

v1docq47 committed:
updated russian translate

- renamed file 'MoneroAddressesCheatsheet20201206.docx' and 'MoneroAddressesCheatsheet20201206.pdf' to 'MoneroAddressesCheatsheet20201206-RU.docx' and 'MoneroAddressesCheatsheet20201206-RU.pdf'
- added 'Translate by v1docq47'
- some typos fix

v1docq47 commented:
Thanks for your comments! ;)
I corrected the position of the arrows and added the note "Перевод шпаргалки выполнил v1docq47".

In addition, I decided to abandon the idea of adding an avatar (I don't have a suitable file) and limited myself to a footnote and link to my GitHub repository:)

baro77 commented:
Thanks for the fixes! I have just squashed & merged it, and added a references in the opening README.md.
Feel free to publish it wherever you want, and if you want I'll also add a reference to the copy on your forum when available
